### DISCUSSION TITLE
Is it possible to not permit select a range if disabled days are in range ?

### DISCUSSION THREAD

**Discussion by vilija19** [dmuy/duDatepicker#38] [Q&A] [answered]
Hello.
Nice picker. Thanks.

If you select a range (for example 10-20) with disabled days (for example 13-14), the script will allow you to confirm this range.
But is the problem because 13-14 already booked.
How to resolve it?

**Comment by dmuy** [comment, marked as answer]
Implemented on duDatepicker v2.0.3
